plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/protocol/OpcuaProtocolLogic.java

```diff
@@ -195,7 +195,7 @@ public CompletableFuture<PlcReadResponse> read(PlcReadRequest readRequest) {
                         future.complete(new DefaultPlcReadResponse(request, status));
                         return;
                     }
-                } catch (ParseException e) {
+                } catch (Exception e) {
```

Comment:
```suggestion
                } catch (ParseException|PlcRuntimeException e) {
```
This would be a better approach then (e.g Exception is also catching InterruptedException)